lib/FileSystemInfo.js

```diff
@@ -1560,6 +1573,11 @@ class FileSystemInfo {
 			}
 		};
 		const checkManaged = (path, managedSet) => {
+			for (const nonManagedPath of this.nonManagedPathsWithSlash) {
+				if (path.startsWith(nonManagedPath)) {
+					return false;
+				}
+			}
```

Comment:
I have a question regarding this behavior, sorry in advance if it sounds silly. 

Though it'd be perfect for our usecase (🙌), is it the usual way of treating include/exclude in the other places of Webpack (for modules rules for example)?

In other words, what is considered _the_ best practice? includes overruling exludes? or excludes overruling includes?